tests/e2e/specs/modules/adsense/setup-new-user.test.js

```diff
@@ -138,7 +140,9 @@ describe( 'setting up the AdSense module', () => {
 
 		await proceedToAdsenseSetup();
 
-		await expect( page ).toMatchElement( '.googlesitekit-setup-module__title', { text: /We’re getting your site ready for ads/i } );
+		await page.waitForSelector( '.googlesitekit-setup-module--adsense' );
+
+		await expect( page ).toMatchElement( '.googlesitekit-heading-4.googlesitekit-setup-module__title', { text: /We’re getting your site ready for ads/i } );
```

Comment:
Is the `.googlesitekit-heading-4` class necessary here? I'd prefer to not add presentational classes to the selector if possible.

Reply:
Oh, good call. It wasn't; I've removed it 👍